2022-08-12 23:05:02,730 INFO    MainThread:1234217 [wandb_setup.py:_flush():76] Loading settings from /home/matteo/.config/wandb/settings
2022-08-12 23:05:02,730 INFO    MainThread:1234217 [wandb_setup.py:_flush():76] Loading settings from /home/matteo/Unsupervised/vaesim_baselines/VAESIM/wandb/settings
2022-08-12 23:05:02,730 INFO    MainThread:1234217 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-08-12 23:05:02,730 INFO    MainThread:1234217 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-08-12 23:05:02,731 INFO    MainThread:1234217 [wandb_init.py:_log_setup():428] Logging user logs to /home/matteo/Unsupervised/vaesim_baselines/VAESIM/wandb/run-20220812_230502-3upbb0by/logs/debug.log
2022-08-12 23:05:02,731 INFO    MainThread:1234217 [wandb_init.py:_log_setup():429] Logging internal logs to /home/matteo/Unsupervised/vaesim_baselines/VAESIM/wandb/run-20220812_230502-3upbb0by/logs/debug-internal.log
2022-08-12 23:05:02,731 INFO    MainThread:1234217 [wandb_init.py:_jupyter_setup():378] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f1cd9fd4b80>
2022-08-12 23:05:02,732 INFO    MainThread:1234217 [wandb_init.py:init():462] calling init triggers
2022-08-12 23:05:02,732 INFO    MainThread:1234217 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {'dataset': 'MNIST10', 'type': 'VAESIM', 'latent_dim': 32, 'n_basis': 40, 'input_dim': (1, 32, 32), 'epochs': 20, 'BS': 2000, 'init_lr': 0.0001, 'sample_cluster': False, 'use_kl_loss': True, 'use_similarity_loss': True, 'reinitialization_probability': 0.0, 'kl_weight': 0.005, 'sim_weight': 0.1}
2022-08-12 23:05:02,732 INFO    MainThread:1234217 [wandb_init.py:init():515] starting backend
2022-08-12 23:05:02,732 INFO    MainThread:1234217 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-12 23:05:02,754 INFO    MainThread:1234217 [backend.py:ensure_launched():217] starting backend process...
2022-08-12 23:05:02,771 INFO    MainThread:1234217 [backend.py:ensure_launched():222] started backend process with pid: 1237633
2022-08-12 23:05:02,774 INFO    MainThread:1234217 [wandb_init.py:init():525] backend started and connected
2022-08-12 23:05:02,802 INFO    MainThread:1234217 [wandb_run.py:_label_probe_notebook():1089] probe notebook
2022-08-12 23:05:02,803 INFO    MainThread:1234217 [wandb_run.py:_label_probe_notebook():1099] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-08-12 23:05:02,803 INFO    MainThread:1234217 [wandb_init.py:init():596] updated telemetry
2022-08-12 23:05:02,806 INFO    MainThread:1234217 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2022-08-12 23:05:03,913 INFO    MainThread:1234217 [wandb_run.py:_on_init():1924] communicating current version
2022-08-12 23:05:03,963 INFO    MainThread:1234217 [wandb_run.py:_on_init():1928] got version response upgrade_message: "wandb version 0.13.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-12 23:05:03,963 INFO    MainThread:1234217 [wandb_init.py:init():662] starting run threads in backend
2022-08-12 23:05:07,930 INFO    MainThread:1234217 [wandb_run.py:_console_start():1898] atexit reg
2022-08-12 23:05:07,931 INFO    MainThread:1234217 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.WRAP
2022-08-12 23:05:07,932 INFO    MainThread:1234217 [wandb_run.py:_redirect():1808] Wrapping output streams.
2022-08-12 23:05:07,935 INFO    MainThread:1234217 [wandb_run.py:_redirect():1832] Redirects installed.
2022-08-12 23:05:07,935 INFO    MainThread:1234217 [wandb_init.py:init():688] run started, returning control to user process
2022-08-12 23:05:07,946 INFO    MainThread:1234217 [wandb_init.py:_pause_backend():341] pausing backend
2022-08-12 23:05:07,946 INFO    MainThread:1234217 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-12 23:05:07,955 INFO    MainThread:1234217 [wandb_init.py:_resume_backend():350] resuming backend
2022-08-12 23:05:12,463 INFO    MainThread:1234217 [wandb_init.py:_pause_backend():341] pausing backend
2022-08-12 23:05:12,463 INFO    MainThread:1234217 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-12 23:05:12,470 INFO    MainThread:1234217 [wandb_init.py:_resume_backend():350] resuming backend
2022-08-12 23:05:13,490 INFO    MainThread:1234217 [wandb_init.py:_pause_backend():341] pausing backend
2022-08-12 23:05:13,490 INFO    MainThread:1234217 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-12 23:05:13,494 INFO    MainThread:1234217 [wandb_init.py:_resume_backend():350] resuming backend
2022-08-12 23:05:13,506 INFO    MainThread:1234217 [wandb_init.py:_pause_backend():341] pausing backend
2022-08-12 23:05:13,508 INFO    MainThread:1234217 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-12 23:05:13,513 INFO    MainThread:1234217 [wandb_init.py:_resume_backend():350] resuming backend
2022-08-12 23:05:14,016 INFO    MainThread:1234217 [wandb_init.py:_pause_backend():341] pausing backend
2022-08-12 23:05:14,017 INFO    MainThread:1234217 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-12 23:13:46,145 INFO    MainThread:1234217 [wandb_run.py:_atexit_cleanup():1867] got exitcode: 0
2022-08-12 23:13:46,147 INFO    MainThread:1234217 [wandb_run.py:_restore():1839] restore
2022-08-12 23:13:47,040 INFO    MainThread:1234217 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 491
  total_bytes: 491
}

2022-08-12 23:13:47,326 INFO    MainThread:1234217 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 491
  total_bytes: 491
}

2022-08-12 23:13:48,185 INFO    MainThread:1234217 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 491
  total_bytes: 491
}

2022-08-12 23:13:48,287 INFO    MainThread:1234217 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 491
  total_bytes: 16666
}

2022-08-12 23:13:48,392 INFO    MainThread:1234217 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 491
  total_bytes: 16666
}

2022-08-12 23:13:48,494 INFO    MainThread:1234217 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7918
  total_bytes: 16666
}

2022-08-12 23:13:48,599 INFO    MainThread:1234217 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16666
  total_bytes: 16666
}

